spec/models/youtube_profile_spec.rb

```diff
@@ -34,10 +34,26 @@
   describe '#after_create' do
     it 'creates RssFeed and RssFeedUrl' do
       profile = described_class.create!(channel_id: 'my_channel_id',
-                                       title: 'My Awesome Channel')
+                                        title: 'My Awesome Channel')
       expect(profile.rss_feed).to be_present
       expect(profile.rss_feed.rss_feed_urls.first.url).to eq('https://www.youtube.com/channel/my_channel_id')
       expect(profile.rss_feed.navigation).to be_nil
     end
   end
+
+  describe 'imported_today' do
+    context 'when no profiles have been updated' do
+      it 'returns no profiles' do
+        expect(described_class.imported_today).to be_empty
+      end
+    end
+
+    context 'when one has been updated' do
+      before { described_class.first.update!(imported_at: Time.now) }
+
+      it 'returns the updated profile' do
+        expect(described_class.imported_today).to eq([described_class.first])
+      end
+    end
```

Comment:
This spec should be updated to test the scope more explicitly. (The context should also be updated to specify `imported` rather than `updated`.) As currently written, the scope could include all profiles (the fact that the other fixtures in the DB don't include an `imported_at` value is not obvious in the spec). We should clarify that profiles with older `imported_at` values aren't included, i.e.:
```ruby
  describe 'imported_today' do
    context 'when no profiles have been imported' do
      it 'returns no profiles' do
        expect(described_class.imported_today).to be_empty
      end
    end

    context 'when profiles have been imported' do
      before do
        described_class.first.update!(imported_at: Time.now)
        described_class.last.update!(imported_at: 1.day.ago)
      end

      it 'includes profiles that were imported today' do
        expect(described_class.imported_today).to eq([described_class.first])
      end
    end
  end
```

Reply:
Changed.